runelite-client/src/main/java/net/runelite/client/plugins/idlenotifier/IdleNotifierPlugin.java

```diff
@@ -355,7 +362,7 @@ public void onGameTick(GameTick event)
 
 		if (config.animationIdle() && checkAnimationIdle(waitDuration, local))
 		{
-			notifier.notify("[" + local.getName() + "] is now idle!");
+				notifier.notify("[" + local.getName() + "] is now idle!");
```

Comment:
You need to fix your indentation here.

```suggestion
			notifier.notify("[" + local.getName() + "] is now idle!");
```